### DISCUSSION TITLE
Support for sphinx.ext.intersphinx for Ablog

### DISCUSSION THREAD

**Discussion by guillaumerenault** [pradyunsg/furo#153] [Potential Issue] [answered]
Hello,

I'm trying to use [Ablog](https://ablog.readthedocs.io/) with furo.
Following the [getting-started](https://ablog.readthedocs.io/en/latest/#getting-started) instructions with Ablog, the following extensions must be added:
```python
# 1. Add 'ablog' and 'sphinx.ext.intersphinx' to the list of extensions
extensions = [
    '...',
    'ablog',
    'sphinx.ext.intersphinx',
]
```

However when doing this and building the documentation Furo does not seem to support this,
index.html content:
```html
ERROR: This page was generated by inheriting from "layout.html".

Furo (the documentation theme that this site is generated with) does not support this, and has generated this page. If you are the author of this documentation, please look into why this page was generated using the "layout.html" template and fix the root cause.

Debugging information, if you need to report this error somewhere:
Sphinx: 3.5.4
Furo: 2021.04.11.beta34

pagename: index
```

When only adding 'ablog' in the extensions without 'sphinx.ext.intersphinx', it seems to be able to build but I'm guessing some Ablog features will be missing.

Has anyone encountered this problem ?

Thank you for your help

**Comment by pradyunsg** [comment, marked as answer]
Hi there! I suggest reaching out to the ablog project's maintainers, and flagging this to them.

**Comment by ABD-01** [comment]
Hi @guillaumerenault
If you are still looking to use [Ablog](https://ablog.readthedocs.io/) with furo, you can see into [ablog_with_furo](https://abd-01.github.io/ablog_with_furo/)
